### PR TITLE
QA-13567 use the sortBy introduced to sort the useTreeEntries results.

### PR DESCRIPTION
**Pull request by XiangYuHe**
## JIRA

https://jira.jahia.org/browse/QA-13567

## Description

Using the sort by to get the sorted results

Example:
`query test($path: String!, $sorter: InputFieldSorterInput) {
    jcr {
      nodeByPath(path: $path) {
        children(fieldSorter: $sorter) {
          nodes {
            name
          }
        }
      }
    }
}`

`{
  "path":"/sites/digitall/files",
  "sorter":{ "fieldName": "name", "sortType": "ASC"}
}`

### CODE REVIEW

File: src/javascript/JContent/ContentTree/ContentTree.jsx [hunk 1]
```diff
@@ -25,7 +26,8 @@ export const ContentTree = ({lang, siteKey, path, openPaths, setPath, openPath,
         openableTypes: item.config.openableTypes,
         selectableTypes: item.config.selectableTypes,
         queryVariables: {language: lang},
-        hideRoot: item.config.hideRoot
+        hideRoot: item.config.hideRoot,
+        sortBy: SORT_CONTENT_TREE_BY_NAME_ASC
```
Comment (AKarmanov) [inline src/javascript/JContent/ContentTree/ContentTree.jsx:30]: Does this sort pages tree as well and if it does do we want to do this? We also need to have yarn.lock updated with updated version of dependencies.

Comment (XiangYuHe) [reply src/javascript/JContent/ContentTree/ContentTree.jsx:30]: Yes it sorts the pages tree as well
I will check with @fpral on that one
Thanks (I will note it to update the yarn.lock with the new dependency) good catch

Comment (XiangYuHe) [reply src/javascript/JContent/ContentTree/ContentTree.jsx:30]: Updated the package.json and yarn.lock
Waiting on feedback from @fpral

Comment (XiangYuHe) [reply src/javascript/JContent/ContentTree/ContentTree.jsx]: Feed-back to update this portion to exclude for item.key === pages

File: src/javascript/JContent/ContentTree/ContentTree.jsx [hunk 2]
```diff
@@ -17,7 +18,7 @@ export const ContentTree = ({lang, siteKey, path, openPaths, setPath, openPath,
         openPaths.push(rootPath);
     }
 
-    const {treeEntries, refetch} = useTreeEntries({
+    const json = {
```
Comment (AKarmanov) [inline src/javascript/JContent/ContentTree/ContentTree.jsx:21]: I would probably call it something other than json :) but other than that looks fine

**Review by AKarmanov — APPROVED** [review]

File: src/javascript/JContent/ContentTree/ContentTree.jsx (hunk 2 above)
Comment (XiangYuHe) [reply src/javascript/JContent/ContentTree/ContentTree.jsx:21]: The universal json variable is all I can think about haha
Maybe `useTreeOptionsJson` is better?

Comment (AKarmanov) [reply src/javascript/JContent/ContentTree/ContentTree.jsx:21]: Anything that would tie it to the fact that it is used in the useTreeEntries hook is better for me )

Comment (XiangYuHe) [reply src/javascript/JContent/ContentTree/ContentTree.jsx:21]: okie sounds good

**Pull request merged**